saleor/graphql/webhook/mutations.py

```diff
@@ -40,6 +56,10 @@ class WebhookCreateInput(graphene.InputObjectType):
         description="The secret key used to create a hash signature with each payload.",
         required=False,
     )
+    query = graphene.String(
+        description="Subscription query used to define a webhook payload.",
```

Comment:
This should be marked as a preview_feature.

Reply:
done.